seed.js

```diff
@@ -112,92 +129,85 @@ var seedFilms = function () {
     },
     {
         title: 'The Matrix',
-        categories: ['5532c18a0dad5827ab24af49'],
+        categories: ['Thriller'],
         description: 'A computer hacker learns from mysterious rebels about the true nature of his reality and his role in the war against its controllers.',
         price: 299,
         photo: 'http://www.1art1.de/images/imagel/m//m1210.jpg',
         inventory: 8
     }
-    ];
-
-    return q.invoke(Film, 'create', films);
-
-};
+];
 
-
-
-var seedCategories = function () {
-    var categories = [
-        { name: 'Action' }, 
-        { name: 'Adventure' }, 
-        { name: 'Animation' }, 
-        { name: 'Comedy' }, 
-        { name: 'Documentary' }, 
-        { name: 'Drama' }, 
-        { name: 'Fantasy' }, 
-        { name: 'Horror' },
-        { name: 'Musical' },
-        { name: 'Mystery' },
-        { name: 'Romance' },
-        { name: 'Sci-Fi' },
-        { name: 'Sport' },
-        { name: 'Thriller'}
-    ]; 
-
-    return q.invoke(Category, 'create', categories);
-}
-
-connectToDb.then(function () {
-    var promisesForSeeds = []; 
-    // users
-     var userPromise = getCurrentUserData().then(function (users) {
-        if (users.length === 0) {
-            return seedUsers();
-        } else {
-            console.log(chalk.magenta('Seems to already be user data, exiting!'));
-            return;
-        }
-    }).then(function (users) {
-        if (users) {
-          console.log(chalk.green('User seed successful!'));
-        }
-    });
-
-    // films
-     var filmPromise = getCurrentFilmData().then(function (films) {
-        if (films.length === 0) {
-            return seedFilms();
-        } else {
-            console.log(chalk.magenta('Seems to already be film data, exiting!'));
-            return;
-        }
-    }).then(function (films) {
-        if (films) {
-          console.log(chalk.green('Film seed successful!'));
-        }
+var seedReviews = [
+    // seed users and films by name
+    {
+        user: "Obama", 
+        date: new Date (2014, 7, 27, 16, 54, 59),
+        comment: "This movie is cool",
+        rating: 4,
+        film: "Die Hard"
+    },
+    {
+        user: "Obama", 
+        date: new Date (2014, 7, 20, 10, 4, 20),
+        comment: "This movie is terrible",
+        rating: 1,
+        film: "The Matrix"
+    }
+]; 
+
+var createFilms = 
+    connectToDb.then(function () {
+        return q.invoke(Category, 'create', seedCategories);
+    })
+    .then(function () {
+        return Category.find().exec();
+    })
+    .then(function (categories) {
+        console.log(chalk.green('Category seed successful!'));
+        seedFilms.forEach(function (film, filmIndex, filmArr) {
+            film['categories'].forEach(function (elem, index, arr) {
+                var filmID = findIdFromDb(elem, categories, 'name');
```

Comment:
should this be 'categoryId'? That's what's getting returned here, right? Trying to follow chain...